openreview/venue/raw_templates.py

```diff
@@ -0,0 +1,483 @@
+                'readers': ['${../../content.venueid.value}', '${../../content.venueid.value}/${../../content.name.value}\\${note.number}/Authors'], ## note.number needs to be escaped. It is defined at note creation
+                'writers': ['${../..content.venueid.value}'],
+                'note': {
+                    'id': {
+                        'param': {
+                            'withInvitation': '${edit.invitation}', ## can I reference the invitation id of the edit
+                            'optional': True
+                        }
+                    },
+                    'signatures': ['${../../../content.venueid.value}/${../../../content.name.value}\\${note.number}/Authors'],
+                    'readers': ['${../../../content.venueid.value}', '${../../../content.venueid.value}/${../../../content.name.value}\\${note.number}/Authors'],
+                    'writers': ['${../../../content.venueid.value}', '${../../../content.venueid.value}/${../../../content.name.value}\\${note.number}/Authors'],
+                    'content': {
+                        'title': {
+                            'order': 1, ## we store this value in the invitation edit but we don't store it in the note, should we assume that only 'value' and 'readers' will be saved in the note content?
+                            'type': 'string', 
+                            'value': { 
+                                'param': { 
+                                    'description': 'Title of paper. Add TeX formulas using the following formats: $In-line Formula$ or $$Block Formula$$', ## Same here.
+                                    'regex': '.{1,250}' 
+                                } 
+                            }
+                        },
+                        'authors': {
+                            'order': 2,
+                            'type': 'string[]', 
+                            'value': { 
+                                'param': { 
+                                    'description': 'Comma separated list of author names.',
+                                    'regex': '[^;,\\n]+(,[^,\\n]+)*' 
+                                },
+                               'hidden': True
+                            },
+                            'readers': [ '${../../../../content.venueid.value}', '${../../../../content.venueid.value}/${../../../../.content.name.value}\\${note.number}/Authors']
+                        },
+                        'authorids': {
+                            'order': 3,
+                            'type': 'group[]',
+                            'value': { 
+                                'param': { 
+                                    'description': 'Search author profile by first, middle and last name or email address. If the profile is not found, you can add the author completing first, middle, last and name and author email address.',
+                                    'regex': r'~.*' 
```

Comment:
This seems unnecessary, but it's needed because type is outside now :(. Previously, passing the type was enough